src/libraries/System.Runtime.InteropServices/gen/DllImportGenerator/DllImportGenerator.cs

```diff
@@ -143,34 +126,24 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                     Environment = data.Right
                 })
                 .Select(
-                    (data, ct) =>
-                    {
-                        IncrementalTracker?.RecordExecutedStep(new IncrementalityTracker.ExecutedStepInfo(IncrementalityTracker.StepName.CalculateStubInformation, data));
-                        return (data.Syntax, StubContext: CalculateStubInformation(data.Symbol, data.Environment, ct));
-                    }
+                    static (data, ct) => (data.Syntax, StubContext: CalculateStubInformation(data.Symbol, data.Environment, ct))
                 )
                 .WithComparer(Comparers.CalculatedContextWithSyntax)
+                .WithTrackingName(StepNames.CalculateStubInformation)
```

Comment:
For my edification, what does WithTrackingName do?

Reply:
`WithTrackingName` tells the generator driver to mark the step information about executions of this step (the `Select` call) with this name when step tracking is enabled. When a GeneratorDriver with step tracking enabled executes this generator, the `TrackedSteps` property on the `GeneratorRunResult` with the results from this generator will have a key with the provided name and all of the executions of this step will be recorded in the value for that key in the dictionary.